exercises/git-ignore/files/file1.txt

```diff
@@ -1 +0,0 @@
-Do not gitginore this file.
```

Comment:
This says that this file should not be gitignored, why is this ignored then?

Reply:
I have updated the pattern in .gitignore . Now its not ignored.